lib/util/serialize-chunks.d.ts

```diff
@@ -0,0 +1,3 @@
+declare function serializeChunks(chunks: Array<string | number>): string
```

Comment:
Does `Chunk` has a specific definition like `Code` or it can be any `string | number`? 

According to [this function](https://github.com/micromark/micromark/blob/2.9.2/lib/preprocess.js#L52), it seems that `Chunk` is a subset of `Code` but I'm not sure.

Reply:
It is an array of Code or string. The EOF (null) code never occurs!